dashboard/test/integration/routes_test.rb

```diff
@@ -11,7 +11,7 @@ def test_section_student_script_ids_routes
   end
 
   def test_dance_session_cookie_and_cache_headers
-    script = Script.find_by_name('dance')
+    script = create :script, name: 'dance'
```

Comment:
We were only generating a Script fixture for dance before and not all the script_levels and such. Instead we can just make the script inline with factories.